flake8_django/checkers/base_model_checker.py

```diff
@@ -10,11 +10,12 @@ class BaseModelChecker(Checker):
 
     model_name_lookup = None
 
-    def _is_abstract_and_set_to_true(self, element):
+    @staticmethod
+    def _is_abstract_and_set_to_true(element):
         return (
                 isinstance(element, ast.Assign)
-                and element.value.value is True
                 and any(target.id == 'abstract' for target in element.targets)
+                and element.value.value is True
```

Comment:
@avallbona wondering, why did you change the order of the checks? I think we could check for value and avoid doing that for through the target elements first, but maybe there is a reason for this change.

Reply:
@rocioar when you have a model that does not have the abstract attribute and instead has a "verbose_name" attribute when you check if "element.value.value is True" the "element.value" is a string and does not have the attribute value, then if fails saying:

`AttributeError: 'Str' object has no attribute 'value'`

For this I've changed the order, first check if we have an attribute called "abstract" and only if we have it check if its value is True.

Reply:
Oh I see good catch, let's check for NameConstant then first:

This should work:

```python
return (
    isinstance(element, ast.Assign)
    and any(target.id == 'abstract' for target in element.targets)
    and isinstance(element.value, ast.NameConstant)
    and element.value.value is True
)
```

Reply:
done. And added an extra check into the iteration in order to avoid the case when, in the meta class you have a tuple assignation, as you pointed before.

Reply:
I think you will also have to fix the identantion, it's 8 characters now.

Reply:
@avallbona you were probably not seeing this comment because it was marked as resolved, please check identation on this lines.

Reply:
@rocioar fixed the indentation issue.
Thanks for the review.